ironfish/src/memPool/memPool.ts

```diff
@@ -63,7 +63,7 @@ export class MemPool {
     )
 
     this.expirationQueue = new PriorityQueue<ExpirationMempoolEntry>(
-      (t1, t2) => t1.expiration < t2.expiration,
+      (t1, t2) => (t1.expiration === 0 ? false : t1.expiration < t2.expiration),
```

Comment:
Should we add transactions with an expiration of 0 to this queue at all?

Reply:
Yea that's a good point. I think you're right we can safely just not add them to the queue